README.md

```diff
@@ -1,5 +1,11 @@
 ![Ginkgo](/assets/logo.png)
 
+[![Build status](https://gitlab.com/ginkgo-project/ginkgo-public-ci/badges/develop/build.svg)](https://github.com/ginkgo-project/ginkgo/commits/develop)
+[![Coverage](https://codecov.io/gh/ginkgo-project/ginkgo/branch/develop/graph/badge.svg)](https://codecov.io/gh/ginkgo-project/ginkgo)
+[![CDash dashboard](https://img.shields.io/badge/CDash-Access-blue.svg)](http://my.cdash.org/index.php?project=Ginkgo+Project)
+[![Documentation](https://img.shields.io/badge/Documentation-latest-blue.svg)](https://ginkgo-project.github.io/ginkgo/doc/develop/)
```

Comment:
Do we have to change these if we have a `master` branch version as well?

I searched for a macro in markdown for the branch name and found [this issue for markup](https://github.com/github/markup/issues/913), however, it is not resolved yet.
Someone suggested a [pre-commit hook](https://stackoverflow.com/questions/18673694/referencing-current-branch-in-github-readme-md/35368808#35368808) to get around this limitation, however, I don't think it would work with our merges into `develop`. Additionally, this script would require us to have Python installed.

I guess, in the end, we need to remember to change this as well when doing the release.

Reply:
Yes we have to do it by hand, I think that is the simplest way of doing that for now. I did not find anything other than what you have written.

Reply:
Should we add it to the release Issue #222?

Reply:
Yes I just did. We also have the same problem in the main `CMakeLists.txt` with the main project tag as we saw today with @pratikvn.